<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>211</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>211</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_connect">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_send">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>211</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>341</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Выполнить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_repeat">
         <property name="text">
          <string>Повтор</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pb_open">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_left">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_right">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>117</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>441</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>441</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="3">
        <widget class="QSpinBox" name="sb_serv4">
         <property name="minimum">
          <number>500</number>
         </property>
         <property name="maximum">
          <number>2500</number>
         </property>
         <property name="value">
          <number>1500</number>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QSpinBox" name="sb_serv5">
         <property name="minimum">
          <number>660</number>
         </property>
         <property name="maximum">
          <number>1300</number>
         </property>
         <property name="value">
          <number>980</number>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QSpinBox" name="sb_serv6">
         <property name="minimum">
          <number>500</number>
         </property>
         <property name="maximum">
          <number>2500</number>
         </property>
         <property name="value">
          <number>1500</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSlider" name="verticalSlider_5">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>500</number>
         </property>
         <property name="maximum">
          <number>2500</number>
         </property>
         <property name="value">
          <number>1500</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSlider" name="verticalSlider_6">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>500</number>
         </property>
         <property name="maximum">
          <number>2500</number>
         </property>
         <property name="value">
          <number>1500</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QSlider" name="verticalSlider_7">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>660</number>
         </property>
         <property name="maximum">
          <number>1300</number>
         </property>
         <property name="value">
          <number>980</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QSlider" name="verticalSlider_8">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>500</number>
         </property>
         <property name="maximum">
          <number>2500</number>
         </property>
         <property name="value">
          <number>1500</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="sb_serv1">
         <property name="minimum">
          <number>500</number>
         </property>
         <property name="maximum">
          <number>2500</number>
         </property>
         <property name="value">
          <number>1500</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="sb_serv2">
         <property name="minimum">
          <number>500</number>
         </property>
         <property name="maximum">
          <number>2500</number>
         </property>
         <property name="value">
          <number>1500</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="sb_serv3">
         <property name="minimum">
          <number>500</number>
         </property>
         <property name="maximum">
          <number>2500</number>
         </property>
         <property name="value">
          <number>1500</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QSlider" name="verticalSlider">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>500</number>
         </property>
         <property name="maximum">
          <number>2500</number>
         </property>
         <property name="sliderPosition">
          <number>1500</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="verticalSlider_2">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>500</number>
         </property>
         <property name="maximum">
          <number>2500</number>
         </property>
         <property name="sliderPosition">
          <number>1500</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb_serv1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb_serv1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb_serv2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb_serv2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb_serv3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb_serv3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb_serv4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb_serv4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb_serv5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb_serv5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sb_serv6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb_serv6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
